src/PHP.php

```diff
@@ -291,6 +292,7 @@ private function configure_site_files() {
 		$site_docker            = new Site_PHP_Docker();
 		$docker_compose_content = $site_docker->generate_docker_compose_yml( $filter );
 		$default_conf_content   = $this->generate_default_conf( $this->cache_type, $server_name );
+		$custom_conf_content    = \EE\Utils\mustache_render( SITE_PHP_TEMPLATE_ROOT . '/config/nginx/custom.conf.mustache', [] );
```

Comment:
Why are you mustache rendering this file for nothing? Why not use `file_get_contents`()? If there is some plan of adding something later on then you can add mustache rendering at that time.